<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditUser</class>
 <widget class="QWidget" name="EditUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>516</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>376</width>
    <height>516</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>520</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>376</width>
    <height>516</height>
   </size>
  </property>
  <widget class="QWidget" name="editUserWidget" native="true">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>3</y>
     <width>370</width>
     <height>510</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>370</width>
     <height>510</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>370</width>
     <height>510</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>370</width>
     <height>510</height>
    </size>
   </property>
   <widget class="QWidget" name="topWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>370</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>370</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>370</width>
      <height>30</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>370</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#topWidget{
	background-color: rgb(240, 241, 245);
}</string>
    </property>
    <widget class="QPushButton" name="closeBtn1">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>7</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	image: url(:/Icon/close2.png);
	border-radius: 0;
	background-color: rgba(255, 255, 255, 0);
}
QPushButton:hover{
	image: url(:/Icon/close.png);
	background-color: rgb(255, 0, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>7</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	image: url(:/Icon/mini2.png);
	border-radius: 0;
	background-color: rgba(255, 255, 255, 0);
}
QPushButton:hover{
	image: url(:/Icon/mini.png);
	background-color: rgb(227, 228, 232);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>7</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>60</width>
       <height>16</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(136, 137, 139);</string>
     </property>
     <property name="text">
      <string>编辑资料</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="toolWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>370</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>370</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>370</width>
      <height>50</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>370</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#toolWidget{
	background-color: rgb(221, 221, 221);
}</string>
    </property>
    <widget class="QPushButton" name="closeBtn2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>15</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size: 12px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	border-radius: 1px;
}
QPushButton:hover{
	background-color: rgb(242, 242, 242);
}</string>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>15</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size: 12px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	border-radius: 1px;
}
QPushButton:hover{
	background-color: rgb(242, 242, 242);
}</string>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>370</width>
      <height>430</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>370</width>
      <height>430</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>370</width>
      <height>430</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>370</width>
      <height>430</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{
	background-color: rgb(255, 255, 255);
}</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(127, 127, 127);
font: 14px &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>昵    称</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(127, 127, 127);
font: 14px &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>签    名</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(127, 127, 127);
font: 14px &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>邮    箱</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(127, 127, 127);
font: 14px &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>性    别</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(127, 127, 127);
font: 14px &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>所在地</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(127, 127, 127);
font: 14px &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>职    业</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(127, 127, 127);
font: 14px &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>手    机</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>290</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
    border-radius: 5px;
    font-size: 13px;
    color: black;
	background-color: rgb(255, 255, 255);
    border: 1px solid #D1D1D1;
}
QLineEdit:hover{
	
	border-color: rgb(69, 165, 255);
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>290</width>
       <height>65</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
    border-radius: 5px;
    font-size: 13px;
    color: black;
	background-color: rgb(255, 255, 255);
    border: 1px solid #D1D1D1;
}
QLineEdit:hover{
	
	border-color: rgb(69, 165, 255);
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>220</y>
       <width>290</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
    border-radius: 5px;
    font-size: 13px;
    color: black;
	background-color: rgb(255, 255, 255);
    border: 1px solid #D1D1D1;
}
QLineEdit:hover{
	
	border-color: rgb(69, 165, 255);
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>260</y>
       <width>290</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
    border-radius: 5px;
    font-size: 13px;
    color: black;
	background-color: rgb(255, 255, 255);
    border: 1px solid #D1D1D1;
}
QLineEdit:hover{
	
	border-color: rgb(69, 165, 255);
}</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
    border-radius: 5px;
    font-size: 13px;
    color: black;
	background-color: rgb(255, 255, 255);
    border: 1px solid #D1D1D1;
}
QComboBox:hover{
	border-color: rgb(69, 165, 255);
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(127, 127, 127);
font: 14px &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>生    日</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
    border-radius: 5px;
    font-size: 13px;
    color: black;
	background-color: rgb(255, 255, 255);
    border: 1px solid #D1D1D1;
}
QComboBox:hover{
	border-color: rgb(69, 165, 255);
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>300</y>
       <width>290</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
    border-radius: 5px;
    font-size: 13px;
    color: black;
	background-color: rgb(255, 255, 255);
    border: 1px solid #D1D1D1;
}
QLineEdit:hover{
	
	border-color: rgb(69, 165, 255);
}</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>290</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
    border-radius: 5px;
    font-size: 13px;
    color: black;
	background-color: rgb(255, 255, 255);
    border: 1px solid #D1D1D1;
}
QComboBox:hover{
	border-color: rgb(69, 165, 255);
}</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>EditUser</receiver>
   <slot>lower()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
